controllers/nexus/resource/networking/manager.go

```diff
@@ -99,7 +97,7 @@ func (m *Manager) GetRequiredResources() ([]resource.KubernetesResource, error)
 			return nil, fmt.Errorf(resUnavailableFormat, "routes")
 		}
 
-		m.log.Debugf("Generating required %s", framework.RouteKind)
+		m.log.Debug("Generating required", "resources", framework.RouteKind)
```

Comment:
gimme a break, too many logs messages to review. I deprecated the `Debugf` method, but then golanglint started to scream at me. :D

Reply:
Yeah, lots of messages, sorry :-( 

We need to have consistent logging so it can be easily parsed with greps, for example. If a single log message is different from other points in the code it wouldn't be easily identified when troubleshooting.